### ISSUE TITLE
dfn lt lowercased

### ISSUE THREAD

**Issue by jyasskin** [w3c/respec#2511]
## Important info

* URL to affected spec: https://www.w3.org/TR/appmanifest/#dom-webappmanifest
* ReSpec version: ReSpec 24.31.0
* [x] I did a "hard refresh", but it's still busted. 

## Description of problem

What happened?: https://www.w3.org/TR/appmanifest/#dom-webappmanifest defines "WebAppManifest" as 

```html
<dfn data-dfn-type="dictionary"
     data-lt="webappmanifest"
     data-export=""
     id="dom-webappmanifest"
     data-idl="dictionary"
     data-title="WebAppManifest"
     data-dfn-for=""><code>WebAppManifest</code></dfn>
```

The [source](https://github.com/w3c/manifest/blob/6851b189ebd73b27eff0876e87d442ecafdeb770/index.html#L1574) was:

```html
    <section data-dfn-for="WebAppManifest" data-link-for="WebAppManifest">
      <h2>
        <dfn>WebAppManifest</dfn> dictionary
      </h2>
```

The `data-lt="webappmanifest"` means that in Bikeshed, I have to use `{{webappmanifest}}` instead of `{{WebAppManifest}}`, which seems wrong. Either omitting the `data-lt` or using `data-lt="webappmanifest|WebAppManifest"` seems better.

**Comment by marcoscaceres** [comment]
Oh whoops! I thought we’d fixed this. Will look into it.

**Comment by marcoscaceres** [comment]
> Either omitting the data-lt or using data-lt="webappmanifest|WebAppManifest" seems better.

Going to omit.

**Comment by marcoscaceres** [comment]
Closed via #2512

**Comment by marcoscaceres** [comment]
@jyasskin, will republish manifest now... might take 12-24 hours for Shepherd to pick it up.
